css/store.css

```diff
@@ -0,0 +1,60 @@
+
+.main{
+    display: flex;
+    margin-left: 150px;
+    gap:100px;
+}
+img{
+    width:710px;
+    height:1235px;
+}
+
+h1{
+    font-weight: 400;
+    font-size: 65px;
+    color: rgb(228, 109, 11);
+}
+form{
+    display: flex;
+    flex-direction: column;
+}
+.name_surname_container{
```

Comment:
This class and `city_postcode_container` have the same style. Why don't you reuse it?